docs/Reference/API-Methods.md

```diff
@@ -17,30 +17,100 @@ Content-Type: application/json
 
 **Request Body**
 
-`payload` : *string* - Base64 encoded payload
+`privacyGroupId` : *string* - ID of the privacy group to delete
 
-`from` : *string*  - Public key of sender
+`from` : *string* - Public key of the Orion node deleting the privacy group
 
-`to` : *array of strings* - List of public keys to receive this payload
+**Returns**
 
+`privacyGroupId` : *string* - ID of the deleted privacy group 
 
-**Returns**
+!!! example
+    ```bash tab="curl HTTP request"
+    curl -X POST http://127.0.0.1:8888/deletePrivacyGroupId \
+      -H 'Content-Type: application/json' \
+      -d '{
+        "privacyGroupId": "68/Cq0mVjB8FbXDLE1tbDRAvD/srluIok137uFOaClOIn4m3Nn0P4jDuRGhWK3AAVdvkqQ==",
+        "from": "negmDcN2P4ODpqn/6WkJ02zT/0w0bjhGpkZ8UP6vARk="
+      }'
+    ```
+   
+    ```json tab="Result"
+    "68/Cq0mVjB8FbXDLE1tbDRAvD/srluIok137uFOaClOIn4m3Nn0P4jDuRGhWK3AAVdvkqQ==" 
+    ```
 
-`key` : *string* - Key used to receive the payload
+## findPrivacyGroupId
+
+Finds all privacy groups for the specified members.
+
+**HTTP Verb**
+POST 
+
+**Headers:**
+Content-Type: application/json
+
+**Request Body**
+
+`addresses` : *array of strings* - Public keys of Orion nodes for which to return privacy groups
 
 !!! example
     ```bash tab="curl HTTP request"
-    curl -X POST http://127.0.0.1:8888/send \
+    curl -X POST http://127.0.0.1:8888/findPrivacyGroupId \
       -H 'Content-Type: application/json' \
       -d '{
-    	"payload": "SGVsbG8sIFdvcmxkIQ==",
-    	"from": "4xanJzyaDPcBVMUSwl/tLp+DbXzd3jF9MKk1yJuyewE=",
-    	"to": ["YE5cJRJYTRO4XFo7yuAi/0K9DwjySGjsHB2YrFPnJXo="]
+      "addresses" : [
+          "A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=",
+          "Ko2bVqD+nNlNYL5EE7y3IdOnviftjiizpjRt+HTuFBs="
+      ]
     }'
+    ``` 
+
+## privacyGroupId
+
+Creates a privacy group with the specified members.
+
+**HTTP Verb**
+POST 
+
+**Headers:**
+Content-Type: application/json
+
+**Request Body**
+
+`addresses` : *array of strings* - Public keys of Orion nodes to include in the privacy group
+
+`from` : *string* - Public key of the Orion node creating the privacy group
+
+`name` : *string* - Name of the privacy group. Optional. 
```

Comment:
`name` and `description` fields are not optional.

Reply:
Fixed